src/azure-cli-core/azure/cli/core/commands/__init__.py

```diff
@@ -195,7 +195,8 @@ def _template_progress(self, correlation_id):  # pylint: disable=no-self-use
                         # don't want to show the timestamp
                         json_val = deploy_values.copy()
                         json_val.pop('timestamp', None)
-                        if deploy_values.get('status value', None) != 'Started':
+                        status_val = deploy_values.get('status value', None)
```

Comment:
So what output is produced when the deployment doesn't provide the "status value"?